src/glass/cypress/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../tsconfig.json",
+  "exclude": [],
+  "include": [
+    "**/*.ts",
+    "plugins/index.js"
+  ],
+  "compilerOptions": {
+    "sourceMap": false,
+    "types": [
+      "cypress"
+    ],
+    "target": "es6"
```

Comment:
I think I need to replace this with `es2015` or `2018`?

Reply:
I think we should remove the `target` property and inherit from the `tsconfig.json` (which is already done in line 2).

Reply:
> Guess I also need to add an "exclude 'cypress'" to glass/tsconfig.json.

Yes, do that please.